components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/public/js/throttling/advanced-throttling/index.js

```diff
@@ -1,33 +1,33 @@
 $(function () {
-    debugger;
     var swaggerClient = new SwaggerClient({
```

Comment:
Move this part to the policy.js.[1] So you don't have to re initialize the swagger client

[1] - https://github.com/wso2/carbon-apimgt/blob/C5/components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/fragments/policy-client/public/js/policy.js#L70